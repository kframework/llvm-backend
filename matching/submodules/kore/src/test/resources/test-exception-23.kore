[]

module TEST-EXCEPTION-23

  symbol a{#b}() : n []

endmodule []
