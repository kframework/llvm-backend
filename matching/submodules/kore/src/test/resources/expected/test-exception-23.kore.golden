Parse error: ../../../test/resources//test-exception-23.kore:5:12:
genericIdRawParser: Invalid first character '#'.
